app/models/feedback_submission.rb

```diff
@@ -1,18 +1,32 @@
-class FeedbackSubmission < ActiveRecord::Base
-  validates :body, presence: true
-  validate :validate_email
+class FeedbackSubmission
+  include NonPersistedModel
 
-  before_validation :strip_email_address, on: :create
+  attribute :body, String
+  attribute :email_address, String
```

Comment:
The `email_address` in pvb2 gets validated with an `EmailChecker` (combination of MX record and Sendgrid).

We need to decide what to do about this as the validations don't cover the email address, which means that a user might get a 500 page if the email address is invalid.

We have a few options to solve this:
- Provide a pvb2 endpoint to validate an email address. Then public would validate the email with the same logic as the api. Could also be used to validate the email address used for requesting a booking since it is a validation that happens in the api (VisitorsStep)

- Stop using `EmailChecker` for feedback submissions in pvb2. This would mean that the validations are the same in public and the api. Downside is that if people give us an email it could be invalid and not be able to get back to them.

- Port `EmailChecker` code to public. Means bringing over all of the Sendgrid code, the ugliest solution imo as we'd have to keep in sync the `EmailChecker` code in both sides.

Any thoughts @mloughran @tatyree? Any other options that I have missed that are worth considering? My preferred option is the first one since it simplifies what goes on by cutting out Sendgrid.